dags/sql/select-sensor_id_count_lookback.sql

```diff
@@ -16,11 +17,24 @@ WITH lookback AS ( --noqa: L045
 
 today AS (
     SELECT
-        COUNT(DISTINCT {{ params.sensor_id_col }}) AS today_count
+        COUNT(DISTINCT {{ params.id_col }}) AS today_count, --noqa: L039
+        ARRAY_AGG(DISTINCT {{ params.id_col }}) AS today_ids --noqa: L039
     FROM {{ params.table }}
     WHERE
         {{ params.dt_col }} >= '{{ ds }} 00:00:00'::timestamp
         AND {{ params.dt_col }} < '{{ ds }} 00:00:00'::timestamp + interval '1 day'
+),
+
+ids_dif AS (
+    SELECT ARRAY_AGG(ids_diff) AS ids_diff
+    FROM (
+        SELECT DISTINCT UNNEST(lb.daily_ids) AS ids_diff
```

Comment:
if we are comparing the distinct ids across the entire period here, I think we can omit creating an array for all days within the date range in the lookback cte

Reply:
nvm we use this calculate the lookback avg 🤔